examples/logging/tutorial-server-prometheus/README.md

```diff
@@ -0,0 +1,157 @@
+        ...
+```
+</details>
+
+The config file instructs the server to create an image classification pipeline. Prometheus logs are declared to be exposed on port `6100`, system logging is turned on, and several data logs have been specified.
+
+Thus, once launched, the Server also exposes two endpoints:
+- port `6100`: exposes the `metrics` endpoint through [Prometheus python client](https://github.com/prometheus/client_python).
+- port `5543`: exposes the endpoint for inference.
+
+To spin up the Server execute:
+```
+deepsparse.server --config server_config.yaml
+```
+
+To validate that metrics are being properly exposed, visit `localhost:6100`. It should contain logs in the specific format meant to be used by the PromQL query language.
+
+## 2. Setup Prometheus/Grafana Stack
+
+For simplicity, we have provided `docker-compose.yaml` that spins up the containerized Prometheus/Grafana stack. In that file, we instruct `prometheus.yaml` (a [Prometheus config file](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)) to be passed to the Prometheus container. Inside `prometheus.yaml`, the `scrape_config` has the information about the `metrics` endpoint exposed by the server on port `6100`.
+
+<details>
+    <summary>Click to see Docker Compose file looks like</summary>
+
+```yaml    
+# docker-compose.yaml
+    
+version: "3"
+
+services:
+  prometheus:
+    image: prom/prometheus
+    extra_hosts:
+      - "host.docker.internal:host-gateway"     # allow a direct connection from container to the local machine
+    ports:
+      - "9090:9090" # the default port used by Prometheus
+    volumes:
+      - ${PWD}/prometheus.yaml:/etc/prometheus/prometheus.yml # mount Prometheus config file
+
+  grafana:
+    image: grafana/grafana:latest
+    depends_on:
+      - prometheus
+    ports:
+      - "3000:3000" # the default port used by Grafana
+
+```
+</details>
+
+<details>
+    <summary>Click to see Prometheus Config file looks like</summary>
+    
+```yaml
+# prometheus.yaml
+    
+global:
+  scrape_interval: 15s                      # how often to scrape from endpoint
+  evaluation_interval: 30s                  # time between each evaluation of Prometheus' alerting rules
+
+scrape_configs:
+  - job_name: prometheus_logs               # your project name
+    static_configs:
+      - targets:
+          - 'host.docker.internal:6100'     # should match the port exposed by the PrometheusLogger in the DeepSparse Server config file 
+```
+</details>
+
+To start up a Prometheus stack to monitor the DeepSparse Server, run:
+
+```bash
+cd docker
+docker-compose up
+```
+
+## 3. Launch the Python Client and Run Inference
+
+`client.py` is a simple client, that periodically sends requests to the Server. The client simulates the behavior of some application, 
+that sends the raw inputs to the inference server and receives the outputs. 
+
+Run the following to start inference:
+
+```bash
+python client/client.py client/piglet.jpg 5543
```

Comment:
This was just some stupid image from the internet, just saying :) Maybe let's make sure we do not infringe any copyrights.